tests/foreman/api/test_role.py

```diff
@@ -1792,3 +1792,57 @@ def test_negative_assign_org_admin_to_ldap_user_group(self):
 
         :CaseAutomation: notautomated
         """
+
+
+class RoleSearchFilterTestCase(APITestCase):
+    """
+    Tests adding additional search filters to role filters.
+    """
+
+    @stubbed
+    @tier1
+    def test_positive_role_lce_search(self):
+        """Test role with search filter using lifecycle enviroment.
+
+        :id: 7147f402-8cc0-4f6a-affe-5a0dd122a05a
+
+        :steps:
+            1. Created Lifecycle Environments DEV
+            2. Created Host dev1 in DEV lifecycle environment.
+            3. Created role 'integrationsrole' role:
+                - see https://access.redhat.com/articles/3359731
```

Comment:
can you remove the reference you access.redhat.com?

Reply:
This is a public article. It's also a good reference which is why i included it.